lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -172,6 +185,8 @@ export class RefreshTokenClient extends BaseClient {
      * @param request
      */
     private async createTokenRequestBody(request: CommonRefreshTokenRequest): Promise<string> {
+        const correlationId = request.correlationId || this.config.cryptoInterface.createNewGuid();
```

Comment:
CorrelationId is a required parameter here, you should be able to do without the default

Reply:
Yes, exactly